content/docs/user-guide/data-management/index.md

```diff
@@ -0,0 +1,162 @@
+# Data Management with DVC
+
+DVC changes the way you work with datasets and ML models in order to enable data
+[versioning] and reproducibility, among other
+[benefits](#benefits-and-implications). Let's look at how the workflow evolves.
+
+<cards>
+
+<card heading="Traditional">
+
+```cli
+$ aws s3 cp s3://d/v0 .
+$ python cleanup.py
+$ python train.py
+
+$ gsutil cp gs://d/v1 .
+$ python cleanup.py
+$ python train.py
+
+$ az storagesync ... v2
+$ python cleanup.py
+$ python train.py
+```
+
+</card>
+
+<card heading="With DVC">
+
+```cli
+$ git checkout v0
+$ dvc checkout
+$ dvc pull -r myS3
```

Comment:
Maybe we avoid `git` here and just `dvc pull` from different `--remote` / `.dvc` file combos.